templates/feed.html

```diff
@@ -122,6 +122,7 @@
 
      <span class="donation_amount" style="display:block; text-align: center">$<input type="value" name="donation_amount" required></span>
 
+<!--   Where is the button supposed to show up?      -->
```

Comment:
since this is directly referencing the stripe API, they do a lot of the work. I style it a bit in css and just within the html (ie <class="container blah blah" style="button-type:"circle">) (obvs that's very very pseudo codey but you get it) The stripe button shows up naturally. And since we are still within the jinja2 for loop, this will reference the description of each request, thus changing the pop up info for each request. The button for stripe and all the other info is kinda just sewn into this code and stripe does a lot of the work. For this, since we don't have as _much_ freedom with the html code, I stylize the button within the html. I didn't publish that to the github but I have played around with button styles.